docs/appendix/zowe-yaml-configuration.md

```diff
@@ -512,6 +538,18 @@ These configurations can be used under the `components.caching-service` section:
  Specifies eviction strategy to be used when the storage size is achieved.
 - **`storage.vsam.name`**  
  Specifies the data set name of the caching service VSAM data set.
+- **`storage.infinispan.initialHosts`**
+
+  This property specifies the list of cluster nodes (members). In case of multiple instances, the value for each Caching Service instance can be either a list of all the members, separated by a comma, or just the replica. The format is `${haInstance.hostname}[${zowe.components.caching-service.storage.i˚nfinispan.jgroups.port}]`.  either a list of all the members, separated by a comma, or just the replica. The format is `${haInstance.hostname}[${zowe.components.caching-service.storage.infinispan.jgroups.port}]`.
```

Comment:
The phrase "either a list of all the members, separated by a comma ... " is repeated twice in this definition. It looks like a copy/paste error. Can you confirm?